runtime-image/test/test.ts

```diff
@@ -4,18 +4,18 @@ import * as Docker from 'dockerode';
 import * as path from 'path';
 import * as request from 'request';
 
-const tar: {pack: (dir: string) => any} = require('tar-fs');
+const tar: {pack: (dir: string) => {}} = require('tar-fs');
```

Comment:
In cases where we lack `@types/` npm modules, can we make it a point to open a tracking bug in this repo to go add it?  Feels like the "right thing to do".

Reply:
Yes thats a great idea.